src/common/src/session_config/mod.rs

```diff
@@ -94,6 +94,10 @@ pub struct ConfigMap {
     #[parameter(default = true, rename = "rw_batch_enable_sort_agg")]
     batch_enable_sort_agg: bool,
 
+    /// Enable distributed dml, so a insert, delete and update statement could be executed in distributed way (e.g. running in multiple compute nodes).
+    #[parameter(default = false, rename = "batch_enable_distributed_dml")]
```

Comment:
Let's do some performance test and make default to true.

Reply:
> make default to true

It comes with the cost for no atomicity guarantee. 😕

Reply:
If we support reading external sources (e.g. iceberg) in the future, we can detect it from the plan and auto-enable it.